### PR TITLE
Adds user model

### PR DESCRIPTION
**Pull request by mdesanti**
# Summary
Adds user model with a many to many relationship to books.

# Screenshots 
App up and running:
image

# Trello Card 
https://trello.com/c/mQaVBT1d/21-crear-modelo-de-user

### CODE REVIEW

File: src/main/java/wolox/training/models/User.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  @Column(nullable = false)
+  private String username;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthdate;
+
+  @ManyToMany(cascade = CascadeType.ALL)
```
Comment (zeminlu) [inline src/main/java/wolox/training/models/User.java]: This cascade is wrong. The books exist independantly of the user, and a book could be asociated to more than one user. Do you remember what CascadeType combination is suitable for collections where the only thing you want is to mantain the jointable updated?

File: src/main/java/wolox/training/models/User.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  @JoinTable(name = "books_users",
```
Comment (zeminlu) [inline src/main/java/wolox/training/models/User.java]: Only necessary if you want custom table and column names, given a ManyToMany infers a JoinTable already.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Just two things!

File: src/main/java/wolox/training/models/User.java (hunk 2 above)
Comment (mdesanti) [reply src/main/java/wolox/training/models/User.java]: 👌

File: src/main/java/wolox/training/models/User.java (hunk 1 above)
Comment (mdesanti) [reply src/main/java/wolox/training/models/User.java]: You're right!

**Review by zeminlu — APPROVED** [review]

**Pull request merged**